mtgjson5/set_builder.py

```diff
@@ -298,21 +276,22 @@ def parse_legalities(sf_card_legalities: Dict[str, str]) -> MtgjsonLegalitiesObj
     return card_legalities
 
 
-def parse_rulings(rulings_url: str) -> List[MtgjsonRulingObject]:
+def parse_rulings(oracle_id: Optional[str]) -> List[MtgjsonRulingObject]:
```

Comment:
str | None over Optional (I'll probably do a wide sweep of this in the near future)